core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -25,7 +25,7 @@ import { UrlService } from 'services/contextual/url.service';
 import { UtilsService } from 'services/utils.service';
 
 import { AppConstants } from 'app.constants';
-const hashes = require('hashes.json');
+const hashes = {};
```

Comment:
Why is this change correct?

Reply:
This change is outdated. It is const hashes = require('hashes.json'); at the moment.